docs/uplink/connect-to-tunnels.md

```diff
@@ -0,0 +1,98 @@
+# Connect to tunnels
+
+The tunnel plugin for the inlets-pro CLI can be used to get connection instructions for a tunnel.
+
+Wether the client needs to be deployed as a systemd service on the customers server or as a kubernetes service, with the cli it is easy to generate connection instructions for these different formats by setting the `--format` flag.
```

Comment:
There's a typo in the first word.

Let's have a bullet list of ways to connect at the top of the page.

CLI command
Systemd
Kubernetes YAML Deployment

Reply:
kubernetes => Kubernetes (it's a name)

Reply:
cli => CLI (it's an abbreviation)